apps/snitch_core/test/domain/shipment_engine_test.exs

```diff
@@ -0,0 +1,91 @@
+defmodule Snitch.Domain.ShipmentEngineTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  import Snitch.Factory
+  import Snitch.{OrderCase, StockCase, ShipmentEngineCase}
+  alias Snitch.Data.Schema.{Order, StockLocation}
+  alias Snitch.Domain.ShipmentEngine
+
+  @sl_manifest %{
+    "default" => %{
+      default: true
+    },
+    "backup" => %{},
+    "origin" => %{}
+  }
+
+  setup :variants
+  setup :stock_locations
```

Comment:
AFAICS, not required. This will be needed later if our (default) CSP uses zone info -- and when that happens we have to insert many more things as well.